Detectors/TOF/workflowIO/src/DigitReaderSpec.cxx

```diff
@@ -35,6 +35,8 @@ void DigitReader::init(InitContext& ic)
   LOG(debug) << "Init Digit reader!";
   auto filename = o2::utils::Str::concat_string(o2::utils::Str::rectifyDirectory(ic.options().get<std::string>("input-dir")),
                                                 ic.options().get<std::string>("tof-digit-infile"));
+  mDelayInMuSec1TF = atoi(ic.options().get<std::string>("delay-1st-tf").c_str()) * 1E6;
```

Comment:
No sure this will work: you say it is a delay in seconds, but convert the string to int (so values <1 will make no sense) then multiply it by 10^6. You should either make option to directly provide time in \mus or as float in seconds to be converted to \mus by scaling by 10^6

Reply:
move atoi to atof. Thanks for spotting this. -> Done